features/ALB999_rail_alignment_value_constraints.feature

```diff
@@ -0,0 +1,17 @@
+@implementer-agreement
+@ALB
+Feature: Rail Alignment value constraints
+
+  Background: Rail alignment
+  
+    Given An IfcAlignment
+      And A relationship IfcRelReferencedInSpatialStructure to IfcAlignment from IfcRail
+    
+  Scenario: Agreement on allowed values for horizontal segment types
+    
+    Given A relationship IfcRelNests from IfcAlignment to IfcAlignmentHorizontal and following that
+    Given A relationship IfcRelNests from IfcAlignmentHorizontal to IfcAlignmentSegment and following that
+    Given Its attribute DesignParameters
+    Given Its attribute PredefinedType
```

Comment:
Did you meant `And` in the last three lines? Or are all four conditions to be interpreted independently of one another?

Reply:
There is no functional difference between that. `and` is just a shortcut for the previously used gherkin keyword. But I agree it should be used consistently (maybe it's actually clearer to never use `and`).